cassdegrees/ui/views.py

```diff
@@ -95,3 +95,66 @@ def sampleform(request):
 
 def create_subplan(request):
     return render(request, 'createsubplan.html')
+
+
+# inspired by the samepleform function creatd by Daniel Jang
+def manage_courses(request):
+    courses = requests.get(request.build_absolute_uri('/api/model/course/?format=json')).json()
+    # If POST request, redirect the received information to the backend:
+    if request.method == 'POST':
+        # hard coded url; only temporary
+        model_api_url = 'http://127.0.0.1:8000/api/model/course/'
```

Comment:
```suggestion
        model_api_url = request.build_absolute_uri('/api/model/course/')
```

This Makes it a relative path, meaning we won't have to change it later (We should probably do this in `sampleform` above too)